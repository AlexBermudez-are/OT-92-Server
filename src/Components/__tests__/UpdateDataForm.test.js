import {render, screen, cleanup, fireEvent} from "@testing-library/react";
import UpdateDataForm from "../Organization/UpdateDataForm";


test("should render nameError", () => {
  render(<UpdateDataForm />);
  const input = screen.getByTestId("nameError");
  fireEvent.change(input, {target: {value: ""}});
  expect(input.value).toBe("");
});

test("should render logoError", () => {
  render(<UpdateDataForm />);
  const input = screen.getByTestId("logoError");
  fireEvent.change(input, {target: {value: ""}});
  expect(input.value).toBe("");
});

test("should render longDescriptionError", () => {
  render(<UpdateDataForm />);
  const input = screen.getByTestId("longDescriptionError");
  fireEvent.change(input, {target: {value: ""}});
  expect(input.value).toBe("");
});

test("should render instagramUrlError", () => {
  render(<UpdateDataForm />);
  const input = screen.getByTestId("instagramUrlError");
  fireEvent.change(input, {target: {value: ""}});
  expect(input.value).toBe("");
});

test("should render facebookUrlError", () => {
  render(<UpdateDataForm />);
  const input = screen.getByTestId("facebookUrlError");
  fireEvent.change(input, {target: {value: ""}});
  expect(input.value).toBe("");
});

