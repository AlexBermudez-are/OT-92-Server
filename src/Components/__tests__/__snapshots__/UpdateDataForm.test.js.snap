// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="mt-5"
  data-testid="div-1"
>
  <h1
    className="text-center"
  >
    Editar datos de la organizacion
  </h1>
  <form
    className="form-container form-group mt-5"
    onSubmit={[Function]}
  >
    <label
      aria-labelledby="name"
      className=""
      htmlFor="name"
    >
      Nombre
    </label>
    <input
      className="mt-2 form-control"
      data-testid="nameError"
      id="name"
      name="name"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    <label
      aria-labelledby="logo"
      htmlFor="logo"
    >
      Logo
    </label>
    <div
      className="d-flex justify-content-between"
    >
      <input
        accept="image/png, image/jpeg"
        className="form-control"
        data-testid="logoError"
        id="logo"
        name="logo"
        onBlur={[Function]}
        onChange={[Function]}
        required={true}
        type="file"
      />
    </div>
    <label>
      Descripcion corta
    </label>
    <label
      aria-labelledby="description"
      htmlFor="longDescription"
    >
       Descripcion larga
    </label>
    <input
      className="form-control"
      data-testid="longDescriptionError"
      id="longDescription"
      name="longDescription"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    <label
      aria-labelledby="instagram"
      htmlFor="linkInstragram"
    >
      Link de Instagram
    </label>
    <input
      className="form-control"
      data-testid="instagramUrlError"
      id="linkInstagram"
      name="linkInstagram"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    <label
      aria-labelledby="facebook"
      htmlFor="linkFacebook"
    >
      Link de Facebook
    </label>
    <input
      className="form-control"
      data-testid="facebookUrlError"
      id="linkFacebook"
      name="linkFacebook"
      onBlur={[Function]}
      onChange={[Function]}
      type="text"
      value=""
    />
    <button
      className="btn btn-primary m-auto"
      htmlFor="submit"
      name="submit"
      style={
        Object {
          "width": "10rem",
        }
      }
      type="submit"
    >
      Cargar
    </button>
  </form>
</div>
`;
